src/content/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations.mdx

```diff
@@ -14,29 +14,10 @@ redirects:
   - /docs/infrastructure/microsoft-azure-integrations
 ---
 
-Our [Microsoft Azure](https://azure.microsoft.com/) integrations allow you to monitor and report data about your Azure services to New Relic, providing a comprehensive view of your entire architecture in one place. 
+New Relic has several offerings for monitoring your [Microsoft Azure](https://azure.microsoft.com) applications and services. Our Azure offerings include: 
 
-<Callout variant="tip">
-This doc covers our Azure monitoring integrations. We also offer integrations to [deploy New Relic within the Azure Marketplace](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native), and [support for Azure with our .NET agent](/docs/apm/agents/net-agent/azure-installation/install-net-agent-azure-web-apps/). 
-</Callout>
+* New Relic in Azure: With our [Azure Native New Relic Service](/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native), you can deploy New Relic from the Azure Marketplace. 
```

Comment:
Maybe we can simplify this a little bit with something like...

"* [Azure Native New Relic Service](link): Deploy New Relic from the Azure Marketplace."

After the colon, can we link directly to the Microsoft Azure Marketplace?

Reply:
i tend to think we put in too many links so i usually try to just point to one thing if I can